add native fee token changes to docs (#738)
OffchainLabs/arbitrum-docs: arbitrum-docs/launch-orbit-chain/orbit-quickstart.md

docker-compose logs -f nitro
```

> ### Appendix B: Depositing ETH/Native token

Reviewer: ```suggestion
### Appendix B: Depositing ETH/native token
```

> If you need to deposit more ETH or native tokens into either your Orbit chain account, run this command on the base directory of the setup script, replacing `0xYourPrivateKey` with the private key of the account that will send the ETH, and `<AMOUNT>` with the amount of ETH or your native token to send:

Reviewer: ```suggestion
If you need to deposit more ETH (or native tokens) into your Orbit chain account, run this command on the base directory of the setup script, replacing `0xYourPrivateKey` with the private key of the originating account, and `<AMOUNT>` with the amount to send:
```
